src/pages/index.astro

```diff
@@ -0,0 +1,69 @@
+---
+import BasicMeta from '../components/BasicMeta.astro'
+import Logo from '../components/logo/GoodbyteGames.vue'
+import GamesList from '../components/GamesList.vue'
+import Noise from '../components/Noise.vue'
+---
+<html lang="en">
+	<head>
+		<BasicMeta/>
+		<title>Goodbyte Games</title>
+		<meta name="theme-color" content="#007944">
+		<style>
+			/* stylelint-disable selector-pseudo-class-no-unknown, no-descending-specificity, selector-list-comma-space-after, max-line-length -- todo: fix in Styleguide */
+			/* stylelint-disable declaration-colon-newline-after, color-function-notation, function-parentheses-newline-inside, value-list-comma-space-after, function-comma-space-after, indentation  -- todo: fix in Styleguide */
+			/* stylelint-disablde value-keyword-case */
```

Comment:
Stylelint is being way too agressive. Only leaving the config in so that we have a use case to work backwards from as we dial down some of these rules.